metals/src/main/scala/scala/meta/internal/builds/BloopInstall.scala

```diff
@@ -132,24 +133,30 @@ final class BloopInstall(
           scribe.info(s"skipping build import with status '${result.name}'")
           Future.successful(result)
         case _ =>
-          scribe.debug("Awaiting user response...")
-          for {
-            userResponse <- requestImport(
-              buildTools,
-              buildTool,
-              languageClient,
-              digest,
-            )
-            installResult <- {
-              if (userResponse.isYes) {
-                runUnconditionally(buildTool, isImportInProcess)
-              } else {
-                // Don't spam the user with requests during rapid build changes.
-                notification.dismiss(2, TimeUnit.MINUTES)
-                Future.successful(WorkspaceLoadedStatus.Rejected)
+          if (
+            userConfig().automaticImportBuild == AutoImportBuildKind.Initial || userConfig().automaticImportBuild == AutoImportBuildKind.All
```

Comment:
Sorry for the late question, but was just wondering if this will only work with Bloop, or other build servers as well (like direct BSP with sbt, mill, etc.)? Asking especially in context of the upcoming changes which would allow to choose the default build server and skip the Bloop in #6121. :slightly_smiling_face:

Reply:
So far as I can see looking through the code only Bloop (and possibly Bazel? https://github.com/scalameta/metals/blob/main/metals/src/main/scala/scala/meta/internal/builds/BazelBuildTool.scala#L155-L156) send a request to the user for an initial import. @kasiaMarek @tgodzik maybe you can confirm I've got this right?

Reply:
Yes and @filipwiech is probably right that in https://github.com/scalameta/metals/pull/6121 it should be changed to be consistent with what we do in Bloop and I should adjust it.

Reply:
Thanks for the update, if it's going to be aligned across the board in @kasiaMarek's PR, then I think it's fine (otherwise I guess it would be nice to at least change the setting description and documentation to mention the Bloop limitation). :+1: